amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpPublishFlow.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.amqp.scaladsl
+
+import akka.Done
+import akka.stream.alpakka.amqp.{AmqpSinkSettings, OutgoingMessage}
+import akka.stream.alpakka.amqp.impl.AmqpPublishFlowStage
+import akka.stream.scaladsl.{Flow, Keep}
+import akka.util.ByteString
+
+import scala.concurrent.Future
+
+object AmqpPublishFlow {
+
+  /**
+   * Scala API: Creates an [[AmqpPublishFlow]] that accepts (ByteString, passthrough) elements.
+   *
+   * This stage materializes to a `Future[Done]`, which can be used to know when the Flow completes, either normally
+   * or because of an amqp failure
+   */
+  def simple[O](settings: AmqpSinkSettings): Flow[(ByteString, O), O, Future[Done]] =
```

Comment:
I believe we could use something more convenient as a tuple and somehow merge the pass-through support directly into `OutgoingMessage` (which I'd like to become `AmqpWriteMessage`). 
But leave that for now, that could become another API-changing PR.